lib/publiccloud/gce.pm

```diff
@@ -91,12 +91,16 @@ sub find_img {
 }
 
 sub upload_img {
-    my ($self, $file) = @_;
+    my ($self, $file, $type) = @_;
```

Comment:
I think we need to give `$type` a default value.
Otherwise we will get a `Use of uninitialized value $type ` if not specified.